### PR TITLE
First draft of PHP Quickstart

### PR DESCRIPTION
**Pull request by stanley-cheung**
PHP Quickstart, for issue #237

### CODE REVIEW

File: docs/quickstarts/php-quickstart.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+1. Run the server
+
+   ```sh
+   $ node greeter_server.js
+   ```
+
+From the `examples/php` directory:
+
+2. In another terminal, run the client
```
Comment (LisaFC) [inline docs/quickstarts/php-quickstart.md]: I'd maybe rejig this a bit, so

In another terminal, from the `examples/php` directory:

2. Run the client:

File: docs/quickstarts/php-quickstart.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+## Generate gRPC code
+
+Next we need to update the gRPC code used by our application to use the new service definition. From the `examples/php` directory:
+
+```
+$ proto-gen-php -i . -o . helloworld.proto
+```
+
+This regenerates `helloworld.php`, which contains our generated client and server classes, as well as classes for populating, serializing, and retrieving our request and response types.
```
Comment (LisaFC) [inline docs/quickstarts/php-quickstart.md]: generated client classes (presumably PHP doesn't have server classes?)

File: docs/quickstarts/php-quickstart.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+
+## Update and run the application
+
+We now have new generated server and client code, but we still need to implement and call the new method in the human-written parts of our example application.
+
+### Update the server
+
+In the same directory, open `greeter_server.js`. Implement the new method like this:
```
Comment (LisaFC) [inline docs/quickstarts/php-quickstart.md]: We haven't rerun the Javascript compiler, have we? Do we need to?

File: docs/quickstarts/php-quickstart.md (hunk 1 above)
Comment (stanley-cheung) [inline docs/quickstarts/php-quickstart.md]: Fixed

File: docs/quickstarts/php-quickstart.md (hunk 2 above)
Comment (stanley-cheung) [inline docs/quickstarts/php-quickstart.md]: Fixed

File: docs/quickstarts/php-quickstart.md (hunk 3 above)
Comment (stanley-cheung) [inline docs/quickstarts/php-quickstart.md]: No need.

File: docs/quickstarts/php-quickstart.md [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+   ```
+
+Congratulations! You've just run a client-server application with gRPC.
+
+## Update a gRPC service
+
+Now let's look at how to update the application with an extra method on the
+server for the client to call. Our gRPC service is defined using protocol
+buffers; you can find out lots more about how to define a service in a `.proto`
+file in [What is gRPC?]() and [gRPC Basics: PHP][]. For now all you need
```
Comment (hsaliak) [inline docs/quickstarts/php-quickstart.md]: Discussed with @zinuga can we drop the reference to [What is gRPC?]() here?

**Comment by hsaliak** [comment]
Left a minor comment, otherwise LGTM.

**Comment by stanley-cheung** [comment]
Fixed

File: docs/quickstarts/php-quickstart.md [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+
+```
+$ proto-gen-php -i . -o . helloworld.proto
+```
+
+This regenerates `helloworld.php`, which contains our generated client classes, as well as classes for populating, serializing, and retrieving our request and response types.
+
+## Update and run the application
+
+We now have new generated server and client code, but we still need to implement and call the new method in the human-written parts of our example application.
```
Comment (LisaFC) [inline docs/quickstarts/php-quickstart.md]: We now have new generated client code,...

Comment (stanley-cheung) [inline docs/quickstarts/php-quickstart.md]: Fixed

File: docs/quickstarts/php-quickstart.md [hunk 6]
```diff
@@ -0,0 +1,203 @@
+---
+layout: docs
+title: PHP Quickstart
+---
```
Comment (hsaliak) [inline docs/quickstarts/php-quickstart.md]: Could you please use the following instead?
```---
bodyclass: docs
headline: PHP Quick Start
layout: docs
sidenav: doc-side-quickstart-nav.html
type: markdown
---```

**Comment by hsaliak** [comment]
Could you please move this from quickstarts/quickstart-php.md to quickstart/php.md (after removing the placeholder that currently exists there).

**Comment by stanley-cheung** [comment]
@hsaliak done with moving the PHP quickstarts. Please review. Thanks!

File: docs/quickstart/php.md [hunk 7]
```diff
@@ -4,4 +4,203 @@ headline: PHP Quick Start
 layout: docs
 sidenav: doc-side-quickstart-nav.html
 type: markdown
```
Comment (hsaliak) [inline docs/quickstart/php.md]: sorry i missed this out in the earlier review, could you please change this to be:
```
---
bodyclass: docs
layout: docs
headline: PHP Quickstart
sidenav: doc-side-quickstart-nav.html
type: markdown
---
```

**Comment by hsaliak** [comment]
Just added 1 comment, LGTM otherwise, feel free to update that and merge.

**Pull request merged**